docs/fetching_credentials/README.md

```diff
@@ -0,0 +1,60 @@
+# Fetching Credentials
+
+As seen [here](https://support.demisto.com/hc/en-us/articles/115002567894), it is possible to integrate with 3rd party credentials 
+vaults for Demisto to use when authenticating with integrations. This article provides an example of such integration.
+
+## Requirements
+
+In order to fetch credentials to the Demisto credentials store, the integration needs to be able to retrieve credential objects 
+in the format of a username and password (key:value).
+
+## Implementation
+
+### isFetchCredentials Parameter
+  
+For this example we are going to look at the HasiCorp Vault integration. The first thing you need to do is add a boolean parameter with the name:
+`isFetchCredentials`(You can give it a different display name). When this parameter is set to true, Demisto will fetch credentials from the integration.
+It would look like something like this: ![image](https://user-images.githubusercontent.com/35098543/53886096-eae09600-4027-11e9-8c2d-a46078c3dcc4.png)
+![image](https://user-images.githubusercontent.com/35098543/53886311-69d5ce80-4028-11e9-9755-08585fecff34.png)
+
+### fetch-credentials Command
+
+When Demisto will try to fetch credentials from the integrations, it will call a command called `fetch-credentials`.
```

Comment:
When Demisto tries to fetch credentials from the integrations, it will call a command called `fetch-credentials`.